driver/src/metrics/stablex_metrics.rs

```diff
@@ -63,13 +65,24 @@ impl StableXMetrics {
         BookType::initialize_gauges(&users);
         registry.register(Box::new(users.clone())).unwrap();
 
+        let solver_time_limit_opts = HistogramOpts::new(
+            "dfusion_service_solver_time_limit",
```

Comment:
Nit: Couldn't this theoretically be computed by the time difference between solution and orders fetched?

Reply:
The difference is that it could be possible for the solver to not abide by the time limit. In this case the metrics would be different.

Reply:
Also I thought it would be nice to have a metric to see how much time we usually give the solver for producing a solution.